packages/dev/inspector-v2/src/components/properties/particles/particleSystemProperties.tsx

```diff
@@ -85,3 +69,36 @@ export const ParticleSystemColorProperties: FunctionComponent<{ particleSystem:
         </>
     );
 };
+
+export const ParticleSystemAttractorProperties: FunctionComponent<{ particleSystem: ParticleSystem }> = (props) => {
+    const { particleSystem: system } = props;
+    const gizmoManager = new GizmoManager(system.getScene()!);

```

Comment:
Shouldn't this be wrapped in a useResource? Otherwise we are re-creating this every render, right?